x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -134,6 +134,8 @@
         "manage_autoscaling",
         Set.of("cluster:admin/autoscaling/*")
     );
+    private static final Set<String> MANAGE_ENROLLMENT_PATTERN = Set.of("cluster:admin/enrollment/*");
```

Comment:
manage_enrollment privilege should imply enroll, so the pattern here should be `"cluster:admin/xpack/security/enrollment/*"` if we're leaving the actions in the security plugin. In any case, the pattern should match ENROLL too.

Reply:
also nit: but there is no reason to have a Pattern for one and not for the other privilege as a separate variable. Let's to the same for both, just for consistency's sake

Reply:
Actually, forgot to change it below, meant to leave it under cluster.